### PR TITLE
Update to Julia v1.0 and MOI v0.8

### PR DESCRIPTION
**Pull request by blegat**

### CODE REVIEW

**Comment by blegat** [comment]
@joehuchette up :)

File: .travis.yml [hunk 1]
```diff
@@ -4,26 +4,16 @@ os:
...
-#script:
-#  - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-#  - julia -e 'Pkg.clone(pwd()); Pkg.build("SDPA"); Pkg.test("SDPA"; coverage=true)'
 addons:
   apt_packages:
     - liblapack-dev
     - libopenblas-dev
 after_success:
   # push coverage results to Coveralls and Codecov
-  - julia -e 'cd(Pkg.dir("SDPA")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder()); Codecov.submit(Codecov.process_folder())'
```
Comment (joehuchette) [inline .travis.yml]: Whoops :)

File: src/MOI_wrapper.jl [hunk 2]
```diff
@@ -24,16 +22,35 @@ mutable struct DSDPSolverInstance <: SDOI.AbstractSDSolverInstance
...
+    optimizer.nconstrs = 0
+    optimizer.blkdims = Int[]
+    optimizer.blk = Int[]
+    optimizer.nlpdrows = 0
+    optimizer.lpdvars = Int[]
+    optimizer.lpdrows = Int[]
+    optimizer.lpcoefs = Cdouble[]
+
+    optimizer.x_computed = true
+    optimizer.y_valid = true
```
Comment (joehuchette) [inline src/MOI_wrapper.jl]: Preexisting: Are these default values for ``x_computed`` and ``y_valid`` reasonable for an empty model?

File: src/MOI_wrapper.jl [hunk 3]
```diff
@@ -203,138 +220,172 @@ function MOI.optimize!(m::DSDPSolverInstance)
...
-        return MOI.Success
+        sol_status = GetSolutionType(m.dsdp)
+        if sol_status == DSDP_PDFEASIBLE
+            return MOI.OPTIMAL
+        elseif sol_status == DSDP_UNBOUNDED
+            return MOI.INFEASIBLE
+        elseif sol_status == DSDP_INFEASIBLE
+            return MOI.DUAL_INFEASIBLE
+        elseif sol_status == DSDP_PDUNKNOWN
+            return MOI.ALMOST_OPTIMAL
```
Comment (joehuchette) [inline src/MOI_wrapper.jl]: Not sure this is the correct status here

File: src/MOI_wrapper.jl (hunk 2 above)
Comment (joehuchette) [reply src/MOI_wrapper.jl]: Looping back here after reading the code, my belief is that ``x_computed=false`` and ``y_valid=true`` are correct defaults.

**Comment by joehuchette** [comment]
Thanks!

**Pull request merged**